buildbot/check.py

```diff
@@ -35,7 +73,9 @@ def main():
                         help="builder directory, which is the directory contains source and build directories")
     parser.add_argument("-s", "--src-dir", metavar="SRC_DIR", help="source directory")
     parser.add_argument("-o", "--obj-dir", metavar="OBJ_DIR", required=True, help="build directory")
-    parser.add_argument("-t", "--test-suite", metavar="TEST_SUITE", required=True, help="check-xxx target")
+    parser.add_argument("-t", "--test-suite", metavar="LIT_TEST_SUITE", required=True, help="check-xxx target")
+    parser.add_argument("--gtest-filter", "--gtest_filter",  metavar="GTEST_FILTER",
+                        help="googletest filter", default="*")
```

Comment:
Align parameter name with the name of the googletest unittests.

Reply:
Thanks to @AlexeySachkov I figured out how LIT integrates with the unittests which makes this change unnecessary and I will remove it.